src/main/kotlin/io/github/inductiveautomation/kindling/log/WrapperLogView.kt

```diff
@@ -47,15 +47,19 @@ class WrapperLogView(
 }
 
 object LogViewer : MultiTool, ClipboardTool {
+    private const val MAX_EXTENSION_INDEX = 20
     override val title = "Wrapper Log"
     override val description = "wrapper.log(.n) files"
     override val icon = FlatSVGIcon("icons/bx-file.svg")
-    override val extensions = listOf("log", "1", "2", "3", "4", "5")
+    override val extensions: List<String> = buildList {
+        add("log")
+        addAll((1..MAX_EXTENSION_INDEX).map { it.toString() })
+    }
 
     override fun open(paths: List<Path>): ToolPanel {
         require(paths.isNotEmpty()) { "Must provide at least one path" }
         val events = paths.flatMap { path ->
-            path.useLines { lines -> LogPanel.parseLogs(lines) }
+            path.useLines(Charsets.ISO_8859_1) { lines -> LogPanel.parseLogs(lines) }
```

Comment:
I'm somewhat leery of this change... it'd be nice to be able to select a charset in the file picker, perhaps, as a 'decoration' (supported in JFileChooser)

Reply:
Being able to select the charset is something we discussed. This is sort of a band aid which works with all of the troublesome wrappers we came across and didn't break anything. 

Based on our testing, this is definitely working. But ideally a charset picker I think is best. We could probably catch the exception and prompt the user to select a different charset

Reply:
There won't be an exception if the charset you assume is wrong, is the thing. Any international user (e.g. a wrapper log from a Japanese system) will load just fine...but some/all of the text will be totally mangled. See [Mojibake](https://en.wikipedia.org/wiki/Mojibake).

Reply:
Charset could now live in preferences, with the default being ISO-8859-1